### PR TITLE
docs: add initial draft to hazards doc

### PR DESCRIPTION
**Pull request by angelaw7**
- add sections 1,2,3,4 and table template for 5

### CODE REVIEW

**Comment by angelaw7** [comment]
relates to #37

File: docs/HazardAnalysis/HazardAnalysis.tex [hunk 1]
```diff
@@ -51,31 +54,64 @@
 
 \pagenumbering{arabic}
 
-\wss{You are free to modify this template.}
-
 \section{Introduction}
 
-\wss{You can include your definition of what a hazard is here.}
+A hazard in the context of this document is any property or condition that may
```
Comment (tan-jason) [inline docs/HazardAnalysis/HazardAnalysis.tex:59]: wondering if we should say anything that would cause a loss in terms of performance or accuracy?

**Review by tan-jason — COMMENTED** [review]
overall lgtm! just left the 1 comment but i'll look more into it tmr too 👍

File: docs/HazardAnalysis/HazardAnalysis.tex (hunk 1 above)
Comment (angelaw7) [reply docs/HazardAnalysis/HazardAnalysis.tex:59]: ooh yeah sure I'll add that

Comment (angelaw7) [reply docs/HazardAnalysis/HazardAnalysis.tex:59]: actually I moved the defn from scope to here, I think it flows better that way

**Comment by chene40** [comment]
looks good 👍

**Pull request merged**